packages/app-accounts/src/DownloadButton.tsx

```diff
@@ -0,0 +1,213 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { Button$Sizes } from '@polkadot/ui-app/Button/types';
+import { BareProps, I18nProps, InputErrorMessage } from '@polkadot/ui-app/types';
+import { KeyringPair$Json } from '@polkadot/util-keyring/types';
+
+import React from 'react';
+import translate from './translate';
+import { Trans } from 'react-i18next';
+import FileSaver from 'file-saver';
+import keyring from '@polkadot/ui-keyring/index';
+import isUndefined from '@polkadot/util/is/undefined';
+
+import AddressMini from '@polkadot/ui-app/AddressMini';
+import Button from '@polkadot/ui-app/Button';
+import Modal from '@polkadot/ui-app/Modal';
+import Unlock from '@polkadot/ui-signer/Unlock';
+
+type State = {
+  address: string,
+  password: string,
+  isPasswordModalOpen: boolean,
+  error?: InputErrorMessage
```

Comment:
`React.ReactNode` - this is the last one.

Repeating what I said in earlier comments - the CALLING component handles the translation, passing the string value (although it can now be a number, div, node, anything) to the underlying component.